jbpm-test-util/src/main/java/org/jbpm/test/persistence/scripts/quartzdialects/QuartzDialectResolver.java

```diff
@@ -30,6 +30,7 @@
     private static Map<String, Dialect> DIALECT_BY_NAME = new HashMap<>();
 
     {
+        registerDialect(new MySQLCustomDialect(), "MariaDB");
         registerDialect(new MySQLCustomDialect(), "MySQL");
```

Comment:
Just a suggestion then as both are sharing some custom dialect, but either ways are fine to me. Up to you.
```suggestion
        registerDialect(new MySQLCustomDialect(), "MySQL", "MariaDB");
```

Reply:
Thanks! done